main.go

```diff
@@ -52,41 +49,45 @@ func main() {
 	versionString := fmt.Sprintf("Akubra (%s version)", version)
 	kingpin.Version(versionString)
 	kingpin.Parse()
-
-	conf, err := config.Configure(*configFile)
+	conf, err := parseConfig(*configFile)
 	if err != nil {
-		log.Fatalf("Improperly configured %s", err)
+		log.Fatalf("Configuration corrupted: %s", err)
 	}
 
-	valid, errs := config.ValidateConf(conf.YamlConfig, true)
-	if !valid {
-		log.Printf("YAML validation - errors: %q", errs)
-		os.Exit(1)
+	if *testConfig {
+		os.Exit(0)
```

Comment:
I don't understand. Why we exit with success when config is defined?

Reply:
It's configuration dry-run ./akubra -c config.yaml -t. Like nginx -t validates changed configs